### DISCUSSION TITLE
Memory leak due to cache tag usage

### DISCUSSION THREAD

**Discussion by bellevue7k-user** [statamic/cms#9723] [Q&A]
Only the page with the code blog below gives this error _on average once a week._

`Allowed memory size of 536870912 bytes exhausted (tried to allocate 169795472 bytes) {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Allowed memory size of 536870912 bytes exhausted (tried to allocate 169795472 bytes) at /home/web...../vendor/laravel/framework/src/Illuminate/Cache/FileStore.php:84)`

Then, clearing the Application Cache and Content Stache fixes it. I think that should be here an usage error . Because it is just an team page and does not consist of very large data.

Relevant code:

```
<div class="l-team__cards">
      {{ collection:team sort="last_name" as="teammember" }} 
          {{ teammember }}
              {{ cache key="team_cache_{id}" }}
                  {{ partial:partials/teamcard }}
              {{ /cache }}
          {{ /teammember }}
      {{ /collection:team }}
</div>
```

cache clearing:

```
 Event::listen(function (EntrySaved $event) {
      if ($event->entry->collectionHandle() === 'team') {
          Cache::forget('team_cache_'.$event->entry->id);
      }
  });
```

**Comment by edalzell** [comment]
Sounds like a bug, please submit an issue

**Reply by bellevue7k-user** [reply]
OkThanks! I submitted an issue.
